lms/static/sass/shared/_header.scss

```diff
@@ -145,6 +145,30 @@ header.global {
           }
         }
       }
+
+      .wrapper-more-actions {
+
+        // Some overrides and customization for the button
+        // in this particular use case
+        .button-more {
+          @include margin-left(-1px); // Don't want a double border on the side with the profile name
```

Comment:
NIT: correct [property order](https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#property-and-declaration-order).